### DISCUSSION TITLE
In version 19 Explorer consumes enormous amounts of memory

### DISCUSSION THREAD

**Discussion by Anixx** [valinet/ExplorerPatcher#602] [General]
![изображение](https://user-images.githubusercontent.com/1271736/147488151-183a4ed5-2ce4-4b09-9736-c008d2233c60.png)

Before, the memory consumption was usually around 9-20 MB, now it is around 290 MB and up to 400 MB at peak.
Everything takes a lot of time to load, including notepad or cmd.exe.

**Comment by valinet** [comment]
And this is for sure caused by ExplorerPatcher because…?

**Comment by Anixx** [comment]
I am not sure. It seems to appear when opening the Open Shell start menu for the second time, but previously there were no problems with the same OpenShell version... Maybe a conflict between EP and OpenShell.

**Reply by valinet** [reply]
And that would be generated because of what…? Idk, the latest pre-release versions are pretty stable for me, I have been trying to polish up and fix any remaining issues to have something to finally release. As I told you before, it would be best to talk based on facts, not impressions…
